hack/update-compatibility.sh

```diff
@@ -8,4 +8,4 @@ if [ -z "${CODEGEN:-}" ];then
   CODEGEN="${TOOLS_OUTPUT}/codegen"
 fi
 
-"${CODEGEN}" compatibility --base-dir "${SCRIPT_ROOT}" -v 2
+"${CODEGEN}" compatibility --base-dir "${SCRIPT_ROOT}" -v 1
```

Comment:
Do we need to have this `-v 1` argument here? Can this be omitted?

Reply:
Default value for the log flag is `-v 0` so yep, I've set this deliberately to `-v 1`